### PR TITLE
N15: Add Camp Misty

### PR DESCRIPTION
**Pull request by ReeCocho**
Part of #320 ("Newsletter 15")

### CODE REVIEW

File: content/posts/newsletter-015/index.md [hunk 1]
```diff
@@ -60,6 +60,20 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Camp Misty]
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-015/index.md:63]: These square brackets don't have a corresponding reference-style link, so they're showing up as literal square brackets in the post - if you look at the [example](https://github.com/rust-gamedev/rust-gamedev.github.io/blob/source/CONTRIBUTING.md#style), there's a corresponding URL at the bottom of the section to go with the title.

Can you either fix this link or remove the square brackets?

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Content looks okay, just a minor issue with the title.

File: content/posts/newsletter-015/index.md (hunk 1 above)
Comment (ReeCocho) [reply content/posts/newsletter-015/index.md:63]: Woops! I'm at work right now, but I'll fix it when I get home.

**Review by ozkriff — APPROVED** [review]
:+1: (Fixed a title link and made a few fmt tweaks for consistency)

**Comment by 17cupsofcoffee** [comment]
The title link still isn't rendering properly.

**Comment by ReeCocho** [comment]
So sorry! Should be fixed with the latest commit.

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by 17cupsofcoffee** [comment]
> So sorry! Should be fixed with the latest commit.

No problem, thank you for the fix!

**Comment by ozkriff** [comment]
> Fixed a title link and made a few fmt tweaks for consistency
> The title link still isn't rendering properly.

Dammit, I forgot to actually push the commit))

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Comment by ReeCocho** [comment]
Thank you for including my game in the newsletter!

**Pull request merged**